README.md

```diff
@@ -1 +1,33 @@
-# zoe
\ No newline at end of file
+# zoe [![CircleCI](https://img.shields.io/circleci/build/github/NudelErde/zoe?logo=circleci)](https://circleci.com/gh/NudelErde/zoe)
+*zoe* is a video game engine that I am working on in my free time. It probably won't surpass *Unity3D* or *Unreal Engine* but where is the fun in using any of these if you can make your own!
+
+I want it to be as optimized as possible but am limited to my free time.
+
+
+## Running
+zoe is still work in progress. You can try running [the sandbox](https://github.com/NudelErde/zoe/releases) but please don't expect anything other than a simple demo.
+
+# Compiling
+I don't know why you would like to compile zoe at its current state, but...
+
+You can compile zoe [as we tell CircleCI to](https://github.com/NudelErde/zoe/blob/master/.circleci/config.yml).
+* Make sure you are using a Linux machine to compile and run zoe
+* Install all dependencies `apt install build-essential cmake libglew-dev libx11-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libgl-dev`
+
+1. Clone the repository with all its submodules (Use `git submodule update --init --recursive` after cloning)
```

Comment:
you could clone with `git clone --recurse-submodules https://github.com/NudelErde/zoe`

Reply:
Added